src/main/java/no/nav/veilarbperson/service/PersonV2Service.java

```diff
@@ -137,20 +139,44 @@ public Familiemedlem mapFamiliemedlem(HentPerson.Familiemedlem familiemedlem, Bo
         );
     }
 
+    private Familiemedlem mapFamiliemedlemUtenFnr(HentPerson.RelasjonUtenIdent personUtenFnr) {
+        return new Familiemedlem()
+                .setFornavn(personUtenFnr.getNavn().getFornavn())
+                .setFodselsdato(personUtenFnr.getFoedselsdato())
+                .setDodsdato(null)
+                .setErEgenAnsatt(false)
+                .setHarVeilederTilgang(true)
+                .setRelasjonsBosted(UKJENT_BOSTED);
+    }
+
     public List<Fnr> hentBarnaFnr(List<HentPerson.ForelderBarnRelasjon> familierelasjoner) {
         return familierelasjoner.stream()
                 .filter(familierelasjon -> "BARN".equals(familierelasjon.getRelatertPersonsRolle()))
                 .map(HentPerson.ForelderBarnRelasjon::getRelatertPersonsIdent)
+                .filter(Objects::nonNull)
                 .map(Fnr::of)
                 .collect(Collectors.toList());
     }
 
     public void flettBarn(List<HentPerson.ForelderBarnRelasjon> forelderBarnRelasjoner, PersonV2Data personV2Data) {
         List<Fnr> barnFnrListe = hentBarnaFnr(forelderBarnRelasjoner);
         List<Familiemedlem> barnInfo = hentFamiliemedlemOpplysninger(barnFnrListe, personV2Data.getBostedsadresse());
+        List<Familiemedlem> barnUtenFnrInfo = hentBarnUtenFnr(forelderBarnRelasjoner);
+        barnInfo.addAll(barnUtenFnrInfo);
+
         personV2Data.setBarn(barnInfo);
     }
 
+    private List<Familiemedlem> hentBarnUtenFnr(List<HentPerson.ForelderBarnRelasjon> forelderBarnRelasjoner) {
+        return forelderBarnRelasjoner.stream()
```

Comment:
this seems bit complicated, why its necessary to check `barn.getRelatertPersonsIdent() == null`, this one also: `.map(HentPerson.ForelderBarnRelasjon::getRelatertPersonUtenFolkeregisteridentifikator)`...

Reply:
@Todo: simplify in next iteration